code/modules/paperwork/fax.dm

```diff
@@ -0,0 +1,157 @@
+/obj/machinery/fax
+	name = "Fax Machine"
+	desc = "Bluespace technologies on the application of bureaucracy."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "fax"
+	density = TRUE
+	power_channel = AREA_USAGE_EQUIP
+	max_integrity = 100
+	pass_flags = PASSTABLE
+	circuit = /obj/item/circuitboard/machine/fax
+	/// The unique ID by which the fax will build a list of existing faxes.
+	var/fax_id
+	/// The name of the fax displayed in the list. Not necessarily unique to some EMAG jokes.
+	var/fax_name
+	/// A reference to an `/obj/item/paper` inside the copier, if one is inserted. Otherwise null.
+	var/obj/item/paper/paper_contain
+	/// Necessary to hide syndicate faxes from the general list. Doesn't mean he's EMAGGED!
+	var/syndicate_network = FALSE
+	/// This is where the dispatch and reception history for each fax is stored.
+	var/list/fax_history = list()
+
+/obj/machinery/fax/Initialize(mapload)
+	. = ..()
+	if (!fax_id)
+		fax_id = SSnetworks.assign_random_name()
+	if (!fax_name)
+		fax_name = "Unregistered fax " + fax_id
+
+/obj/machinery/fax/Destroy()
+	QDEL_NULL(paper_contain)
+	return ..()
+
+/obj/machinery/fax/update_icon_state()
+	if(paper_contain)
+		icon_state = "fax_contain"
+	else
+		icon_state = "fax"
+	return ..()
+
+//Emag does not bring you into the syndicate network, but makes it visible to you.
+/obj/machinery/fax/emag_act(mob/user)
+	if(!(obj_flags & EMAGGED))
+		obj_flags |= EMAGGED
+		playsound(src, 'sound/creatures/dog/growl2.ogg', 50, FALSE)
+		to_chat(user, span_warning("An image appears on [src] screen for a moment with Ian in the cap of a Syndicate officer."))
+
+/obj/machinery/fax/wrench_act(mob/living/user, obj/item/tool)
+	. = ..()
+	default_unfasten_wrench(user, tool)
+	return TOOL_ACT_TOOLTYPE_SUCCESS
+
+/obj/machinery/fax/multitool_act(mob/living/user, obj/item/I)
+	var/new_fax_name = tgui_input_text(user, "Enter a new name for the fax machine.", "New Fax Name", , 128)
+	if(!new_fax_name)
+		return
+	if (new_fax_name != fax_name)
+		if (fax_name_exist(new_fax_name))
+			// Being able to set the same name as another fax machine will give a lot of gimmicks for the traitor.
+			if (syndicate_network != TRUE && obj_flags != EMAGGED)
+				to_chat(user, span_warning("There is already a fax machine with this name on the network."))
+				return
+		fax_name = new_fax_name
+	return
+
+/obj/machinery/fax/attackby(obj/item/item, mob/user, params)
+	if(istype(item, /obj/item/paper))
+		if(!paper_contain)
+			paper_contain = item
+			item.forceMove(src)
+			update_appearance()
+		return
+	return
```

Comment:
```suggestion
	return ..()
```
Because paper is early returning, this is only called when the item we're hitting it with *isnt* paper, in which case we do want it to call parent.